spec/spec_helper.rb

```diff
@@ -28,6 +28,27 @@
   end
 end
 
+RSpec::Matchers.define :json_like do |expected, options={}|
```

Comment:
@fedescarpa @aguspina I am adding this matcher to all project. I think it fits most scenarios where you need to check something that has a json-like structure.